### DISCUSSION TITLE
Do you have plan to upgrade statusline?

### DISCUSSION THREAD

**Discussion by thieung** [fitrh/init.nvim#6] [Ideas]
Do you have any plan to upgrade statusline?
`lualine`'s performance looks like slower than `feline` or `heirline`

**Comment by fitrh** [comment]
I've never thought of any other statusline plugins, but since you mentioned,
I thought I'd try `heirline` before deciding whether to replace lualine or not

About performance, do you have anything like a real life comparison between them?

**Comment by thieung** [comment]
Glad to hear you're interested in `heirline`
About performance, I run `PackerProfile` to check and I always see `lualine` take more time to load than `feline` (~10x slower)
